tests/unit/univariate/test_gaussian_kde.py

```diff
@@ -165,6 +166,19 @@ def test_percent_point_invalid_value(self):
         with self.assertRaises(ValueError):
             instance.percent_point(np.array([2.]))
 
+    def test_percent_point_convergence(self):
+        instance = GaussianKDE()
+        instance.fit(sample_univariate_bimodal())
+        x = instance.percent_point(np.array([0.5]))
+        assert abs(instance.cumulative_distribution(x) - 0.5) < 0.01
+
+    def test_percent_point_boundary_values(self):
```

Comment:
Regression test, if all values are 0.0/1.0 (-inf/inf), it used to throw a random error.